src/plays/netlify-card-game/NetlifyCardGame.jsx

```diff
@@ -0,0 +1,229 @@
+import { getPlayById } from "meta/play-meta-util";
+
+import PlayHeader from "common/playlists/PlayHeader";
+import { useState, useEffect, useRef } from "react";
+
+import "./NetlifyCardGame.scss";
+
+import q1 from "./icons/q1.png";
```

Comment:
Make sure each of the images are less than 500KBs

Reply:
every icon is below 50kb even